GUI/QtQuick/qml/WalletProperties/DeleteWalletWarn.qml

```diff
@@ -68,7 +96,7 @@ ColumnLayout {
 
         CustomButton {
             text: qsTr("Export watching-only wallet")
-            visible: wallet_properties_vm.isHardware || wallet_properties_vm.isWatchingOnly
+            visible: isHW()
```

Comment:
I think this condition should be inverted, at least I do know that WO export of WO and HW wallets is non-sense.

Reply:
visible: wallet_properties_vm.isHardware || wallet_properties_vm.isWatchingOnly

You are right.
I modified the original code as it is, but I'll check in detail.